Editor/SketchServiceInstaller.cs

```diff
@@ -3,10 +3,10 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Reflection;
-using AIR.Flume;
+using Actuator.Flume;
```

Comment:
`Actuator.Flume` doesn't exist. Will cause compilation errors in projects with Flume + Sketch.